changelog/v1.15.0.rst

```diff
@@ -53,7 +53,7 @@ Notable Changes & New Features
 
 - Dashboard Interface Revamped
 - New Thermostat Controller implements ESPHome actions for all available Home Assistant climate actions,
-  climate modes, fan modes, and fan swing modes (#1061)
+  climate modes, fan modes, and fan swing modes (#1105)
```

Comment:
```suggestion
  climate modes, fan modes, and fan swing modes (:esphomepr:`1105`)
```

That automatically creates a link

Reply:
Awesome! Thank you for pointing that out @OttoWinter 😄